src/lexedata/enrich/guess_concepticon.py

```diff
@@ -135,30 +135,33 @@ def create_concepticon_for_concepts(dataset: pycldf.Dataset, overwrite: bool = T
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "wordlist",
-        default="cldf-metadata.json",
+        "--metadata",
+        nargs="?",
         type=Path,
-        help="The wordlist to add Concepticon links to",
+        default="Wordlist-metadata.json",
+        help="Path to the metadata.json. Concepticon Concepts are added to 'parameterTable'"
```

Comment:
What the program does should not be part of the functionality of the switch, but of the AP `description`.

Elsewhere you have

    Path to the metadata.json. The metadata file describes the dataset.

How about always

    Path to the JSON metadata file describing the dataset.

?